demo/web/src/main/notebooks/07 Data Replay.md

```diff
@@ -0,0 +1,72 @@
+# Data Replay
+
+Deephaven excels at handling live data. Integrating historical data into real-time analysis is common in a multitude of fields, including machine learning, validation, modeling, simulation, and forecasting.
+
+Here, we show how to take historical data and play it back as real-time data based on timestamps in a table. This example could be easily extended towards a variety of real-world applications.
```

Comment:
```suggestion
Here, we show how to take historical data and play it back as real-time data based on timestamps in a table. This is useful for demonstration purposes, but this example could be easily extended towards a variety of real-world applications.
```

Reply:
Yes, Andy has helped customers pretty often with data replay - a specific application would help new users imagine the possibilities of DH.